features/DocumentController.feature

```diff
@@ -173,6 +173,46 @@ Feature: Document Controller
       | _id          |
       | "document-1" |
 
+  @mappings
+  Scenario: Search with Koncorde filters and ES clause
+    Given an existing collection "nyc-open-data":"yellow-taxi"
+    And I "create" the following multiple documents:
+      | _id          | body                                                |
+      | "document-1" | { "name": "Melis", "age": 25, "city": "Istanbul" }  |
+      | "document-2" | { "age": 25, "city": "Istanbul" }                   |
```

Comment:
to have more logic test, can you add a name field in "document-2" without an "e" letter ?
In your test, you don't test really the wildcard pattern (because you don't have name field)